app/src/components/map/Cluster.js

```diff
@@ -16,9 +16,14 @@ import VectorSource from 'ol/source/Vector';
 import { asArray } from 'ol/color';
 import { Feature, Overlay } from 'ol';
 import { fromLonLat } from 'ol/proj';
+import GeoJSON from 'ol/format/GeoJSON';
 import { getColor } from './olMapColors';
 import { indicatorClassesIcons } from '../../config/trilateral';
```

Comment:
Is it ok to hard-code `trilateral` here?

Reply:
Within the app, we use `this.baseConfig.indicatorClassesIcons`, where the `baseConfig` is loaded according to the current brand. Since you are already importing the store, you could do something like `const { baseConfig } = store.state.config;` and get the `indicatorClassesIcons` from there - but that only works if the store is already filled, otherwise it will be `null`.